src/components/Comments.jsx

```diff
@@ -1,42 +1,113 @@
 
 import { getCommentsById } from "../utils/api";
+import { postComment } from "../utils/api";
 import { useState, useEffect } from "react";
-
+import CommentCard from "./CommentCard";
 
 const Comments = (props) => {
 const {article_id} = props;
 const [isLoading, setIsLoading] = useState(true);
-const [comments, setComments] = useState([])
+const [comments, setComments] = useState([]);
+const [username, setUserName] = useState('');
+const [body, setBody] = useState('');
+const [comment, setComment] = useState({});
+const [err, setErr] = useState(null);
+
+const submitHandler = (event) => {
+   setErr(null)
+    event.preventDefault();
+  
+    setComment({
+        comment_id: (Date.now()),
+        author: username,
+        created_at: (new Date()).toString(),
+        body: body,
+        votes: 0
+    })
+   
+    postComment(article_id, username, body)
+    .then(() => {
+        
+        alert("Your comment was posted. Congrats!")
+    })
+    .catch(() => {
+        setErr("Somethimg went wrong. Check your username and fill out all fields")
+    })
+}
+
+const usernameHandler = (event) => {
```

Comment:
remove the need to type a username to leave a comment. You can either hard code it, or make that you can only post a comment as a logged in user. This makes it easier to navigate

Reply:
Done